### PR TITLE
T922-024 Add is_inherited_primitive property

### PR DESCRIPTION
**Pull request by danielmercier**

### CODE REVIEW

File: ada/language/ast.py [hunk 1]
```diff
@@ -3586,6 +3586,14 @@ def is_view_of_type(comp_view=T.BaseTypeDecl.entity):
     def is_array_or_rec():
         return Entity.is_array | Entity.is_record_type
 
+    @langkit_property(public=True, return_type=Bool)
+    def is_inherited(p=T.BasicDecl.entity):
+        """
+        Assuming that P is a primitive of Self, return wheather the given
```
Comment (pmderodat) [inline ada/language/ast.py:3592]: whether*

File: ada/language/ast.py [hunk 2]
```diff
@@ -3586,6 +3586,14 @@ def is_view_of_type(comp_view=T.BaseTypeDecl.entity):
     def is_array_or_rec():
         return Entity.is_array | Entity.is_record_type
 
+    @langkit_property(public=True, return_type=Bool)
+    def is_inherited(p=T.BasicDecl.entity):
```
Comment (raph-amiard) [inline ada/language/ast.py:3590]: I'd rather you be even more descriptive in the property name: `is_inherited_primitive`. That will make it clearer without the programmer having to check whether the node this is called on is a type or a field or a primitive or whatever.

**Review by raph-amiard — CHANGES_REQUESTED** [review]
Great work @danielmercier ! Just do the two small cosmetic changes and we'll merge that ASAP.

File: ada/language/ast.py (hunk 2 above)
Comment (danielmercier) [reply ada/language/ast.py]: Agreed, I changed it.

File: ada/language/ast.py (hunk 1 above)
Comment (danielmercier) [reply ada/language/ast.py]: Thanks

**Comment by raph-amiard** [comment]
@danielmercier you forgot to update the test ;)

```
INFO     FAIL            properties__is_inherited_primitive: python3 returned status code 1 (0 expected)
Running: python3 test.py test.adb (cwd=/home/travis/build/AdaCore/libadalang/tmp/properties__is_inherited_primitive)
Status code: 1
Output: 
== test.adb ==
Traceback (most recent call last):
  File "test.py", line 13, in <module>
    if t.p_is_inherited(prim):
AttributeError: 'TypeDecl' object has no attribute 'p_is_inherited'
```

**Review by raph-amiard — APPROVED** [review]

**Pull request merged**